index.js

```diff
@@ -6,15 +6,185 @@ var phantomjs = require('phantomjs');
 var moment = require('moment-timezone');
 var cron = require('node-cron');
 
-router.get('/fetch/koinex', function(req, res, next) {
-    db.fetch.find({}).exec(function(err, data) {
+
+router.get('/get_btc/:start_date/:end_date', function(req, res, next) {
```

Comment:
make a single route like 

/get/:symbol/:start_date/:end_date

and symbol will be btc, eth etc

Reply:
ok sir